RELEASE/scripts/autoscend/quests/level_12.ash

```diff
@@ -1766,7 +1766,7 @@ boolean L12_themtharHills()
 
 boolean LX_obtainChaosButterfly()
 {
-	if(in_bhy() || in_pokefam())
+	if(in_bhy() || in_pokefam() || in_glover())
```

Comment:
should have probably gone with replacing
`if(!is_unrestricted($item[chaos butterfly]))` with
`if(!auto_is_valid($item[chaos butterfly]))`